### PR TITLE
feat: added homepage component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    width: 100%;
+}
+
+/* Nav */
+
+.nav-header{
+    position: sticky;
+    top: 0;
+    height: auto;
+    /* box-shadow: inset 0px -1px 1px #e7ebf0; */
```
Comment (varun1233) [inline style.css:117]: Remove commented code

**Review by Sahil25061999 — APPROVED** [review]
flex is being used alot, you can separate that out and give all the classes that require it together, some minor changes require like comments can be removed. Not major changes can be done later.

**Review by varun1233 — APPROVED** [review]
Just remove commented code if it is not required, apart from that code looks good

**Comment by eakansh125** [comment]
> flex is being used alot, you can separate that out and give all the classes that require it together, some minor changes require like comments can be removed. Not major changes can be done later.

Thanks I will take care of that!

File: style.css (hunk 1 above)
Comment (eakansh125) [reply style.css:117]: sure

**Pull request merged**